src/Parser/LASLoader.js

```diff
@@ -61,6 +62,7 @@ class LASLoader {
         const classifications = new Uint8Array(view.pointCount);
         const pointSourceIDs = new Uint16Array(view.pointCount);
         const colors = getColor ? new Uint8Array(view.pointCount * 4) : undefined;
+        const scanAngles = new Int16Array(view.pointCount);
```

Comment:
Following our previous discussion, Point Data Record Formats (PDRF) 0 to 5 ("legacy" formats as of LAS 1.4 spec) `Scan Angle` is encoded as signed char in a valid range from -90 to +90 (degrees).
Since PDRF 6, `Scan Angle` is encoded as a signed short in a valid range from `-30 000` to `+30 000`. Those values represents scan angles from `-180` to `+180` degrees with an increment of `0.006` degree (which `copc.js` applies).

I think you should:
- Add a notice explaining this
- Change to `Float32` since `copc.js` apply this increment and the values returned cannot be encoded as integers:
```suggestion
        const scanAngles = new Float32Array(view.pointCount);
```